### PR TITLE
2024 09 27 jk qene negusa

### PR DESCRIPTION
**Pull request by karljonaskarlsson**
Another qene from Uppsala, images here: https://www.alvin-portal.org/alvin/imageViewer.jsf?dsId=ATTACHMENT-0414&pid=alvin-record:280997

### CODE REVIEW

**Review by nafisa-valieva — APPROVED** [review]
Nice! Looks good and clear for me.

File: new/LIT7077QeneNegusa.xml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7077QeneNegusa" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ንጉሠ፡ ነገሥት፡ ወልድ፡ አመ፡ ቆመ፡ በዓውደ፡ ተሰይጠ፡ ሰላሳ፤</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Nǝguśa nagaśt wald ʾama qoma ba-ʿawda tasayṭa salāsā (Qǝne of kəbr yəʾəti-type)</title>
```
Comment (CarstenHoffmannMarburg) [inline new/LIT7077QeneNegusa.xml:9]: Even if it is grammatically correct, the manuscript has በዓውደ, so it should be transcribed ba-ʿāwda (not ba-ʿawda).

File: new/LIT7077QeneNegusa.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                <title xml:lang="en" corresp="#t1">The king of kings, the Son, when he stood in the courtyard (where) he was sold for thirty (silver coins)…</title>
+                <editor role="generalEditor" key="AB"/>
```
Comment (CarstenHoffmannMarburg) [inline new/LIT7077QeneNegusa.xml:11]: I think, there must be a comma after courtyard and I suggest to change the conjunction after. As far as I know, Jesus was sold, before he was brought to the court:
The king of kings, the Son, when he stood in the courtyard, **for** he was sold for thirty (silver coins)...

**Review by CarstenHoffmannMarburg — CHANGES_REQUESTED** [review]

File: new/LIT7077QeneNegusa.xml (hunk 1 above)
Comment (karljonaskarlsson) [reply new/LIT7077QeneNegusa.xml:9]: You are right! Thank you

File: new/LIT7077QeneNegusa.xml (hunk 2 above)
Comment (karljonaskarlsson) [reply new/LIT7077QeneNegusa.xml:11]: It's a matter of English grammar whether there should be a comma here or not. My feeling is this is a restrictive relative clause and that a comma should therefore not be used, but maybe there are different rules about this.

I'm not sure about translating the asyndetic (relative?) clause as a reason clause, but if you prefer it, I can add it as an option.

Comment (CarstenHoffmannMarburg) [reply new/LIT7077QeneNegusa.xml:11]: It is up to you. The conjunction "for" is just a suggestion by myself. If you prefer "where", you can also keep that. An alternative translation is not necessary in my opinion.

The same on the comma. If you think, that a comma is not needed here, you can leave it like this.

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Pull request merged**